processor/transform/readme-to-mdx.ts

```diff
@@ -87,7 +87,7 @@ const readmeToMdx = (): Transform => tree => {
     if (!('data' in image)) return;
 
     if ('url' in image) image.data.hProperties.src = image.url;
-    const attributes = toAttributes(image.data.hProperties, imageAttrs);
+    const attributes = toAttributes({ ...image, ...image.data.hProperties }, imageAttrs);
```

Comment:
We put the `alt` attribute directly on the `image` node, because that's how `remark` does it. But we weren't following that shape here.